### ISSUE TITLE
ocr plugin does not work for version 0.2.1

### ISSUE THREAD

**Issue by P4radise** [jlipps/appium-ocr-plugin#15]
After updating the plugin to version 0.2.1 I started to receive the following error when running this plugin.

```
appium --relaxed-security --use-plugins=ocr
[Appium] Attempting to load plugin ocr...
[Appium] Requiring plugin at /Users/username/node_modules/appium-ocr-plugin/build/index.js
[Appium] Could not load plugin 'ocr', so it will not be available. Error in loading the plugin was: Cannot find module '/Users/username/node_modules/appium-ocr-plugin/build/index.js' imported from /opt/homebrew/lib/node_modules/appium/build/lib/extension/extension-config.js
[Appium] Error: Cannot find module '/Users/username/node_modules/appium-ocr-plugin/build/index.js' imported from /opt/homebrew/lib/node_modules/appium/build/lib/extension/extension-config.js
    at finalizeResolution (node:internal/modules/esm/resolve:257:11)
    at moduleResolve (node:internal/modules/esm/resolve:914:10)
    at defaultResolve (node:internal/modules/esm/resolve:1038:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:557:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:525:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:246:38)
    at onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:482:36)
    at TracingChannel.tracePromise (node:diagnostics_channel:337:14)
    at ModuleLoader.import (node:internal/modules/esm/loader:481:21)
    at defaultImportModuleDynamicallyForScript (node:internal/modules/esm/utils:227:31)
    at importModuleDynamicallyCallback (node:internal/modules/esm/utils:249:12)
    at PluginConfig.requireAsync (/opt/homebrew/lib/node_modules/appium/lib/extension/extension-config.js:600:23)
    at /opt/homebrew/lib/node_modules/appium/lib/extension/index.js:53:26
```

I also see that the package.json and package-lock.json files have not been updated to indicate the new version 0.2.1.

**Comment by groooooomit** [comment]
Same issue, any update?

**Comment by groooooomit** [comment]
@P4radise Uninstall 0.2.1 and install 0.2.0 temporary fix this issue

`appium plugin install --source=npm appium-ocr-plugin@0.2.0`

**Comment by P4radise** [comment]
@groooooomit Yes, thanks, I know, but I wish the latest version was fixed :)

**Comment by abaiadse** [comment]
same issue, build folder does not exist.

appium log:
```
2024-11-06 12:07:45:450 [Appium] Requiring plugin at /root/.appium/node_modules/appium-ocr-plugin/build/index.js
2024-11-06 12:07:45:451 [Appium] Could not load plugin 'ocr', so it will not be available. Error in loading the plugin was: Cannot find module '/root/.appium/node_modules/appium-ocr-plugin/build/index.js' imported from /usr/lib/node_modules/appium/build/lib/extension/extension-config.js
2024-11-06 12:07:45:471 [Appium] Error: Cannot find module '/root/.appium/node_modules/appium-ocr-plugin/build/index.js'
```

```
$: ls
README.md	node_modules	package.json	src
```

@jlipps @KazuCocoa fyi.

**Comment by jlipps** [comment]
thanks for the heads up, i added a script to ensure `build` is created before publish. This should be fixed in 0.2.2
